trufflescripts/add_token_pair.js

```diff
@@ -41,14 +43,16 @@ module.exports = async () => {
   const dx = await DutchExchange.at(Proxy.address)
   const eth = await TokenETH.deployed()
   const gno = await TokenGNO.deployed()
+  const owl = await TokenOWL.deployed()
```

Comment:
I think, owl is also proxied, like dx, not sure if anything would break but...

Reply:
yup